core/domain/services/admin/privacy/policy/privacy_policy.template.php

```diff
@@ -0,0 +1,72 @@
+<h2><?php esc_html_e('Event Registration Data', 'event_espresso');?></h2>
+<p><?php esc_html_e('We collect information about you during event registration. This information may include but is not limited to:', 'event_espresso');?></p>
+<ul>
+    <li><?php esc_html_e('Your names', 'event_espresso');?></li>
+    <li><?php esc_html_e('Billing address', 'event_espresso');?></li>
+    <li><?php esc_html_e('Mailing address', 'event_espresso');?></li>
+    <li><?php esc_html_e('Email address', 'event_espresso');?></li>
+    <li><?php esc_html_e('Phone number', 'event_espresso');?></li>
+    <li><?php esc_html_e('Location and traffic data (including IP address and browser type)', 'event_espresso');?></li>
+    <li><?php esc_html_e('Any other details that might be requested from you for the purpose of processing your registration or ticket purchase', 'event_espresso');?></li>
+</ul>
+
+<p><?php esc_html_e('Handling this data also allows us to:', 'event_espresso');?></p>
+<ul>
+    <li><?php esc_html_e('Send you important account/purchase/service information.', 'event_espresso');?></li>
+    <li><?php esc_html_e('Respond to your queries, refund requests, or complaints.', 'event_espresso');?></li>
+    <li><?php esc_html_e('Process payments and to prevent fraudulent transactions. We do this on the basis of our legitimate business interests.', 'event_espresso');?></li>
```

Comment:
> Process payments and to prevent fraudulent transactions.

should maybe be:

> Process payments and prevent fraudulent transactions.

but plz confirm with the original author if this text was supplied to you.

Reply:
I wrote it with Seth, but ya that sounds better